src/backtrace_win32.cc

```diff
@@ -1,8 +1,41 @@
-#include "node.h"
+#define _WIN32_WINNT 0x0600
```

Comment:
@nodejs/platform-windows What’s the right way to do this? That’s basically the only question I have left here…

Reply:
This is actually an interesting question. We have multiple version targeting macros:

Reply:
(shirt-return is not the right key-chord. it's not new line, it's Submit)
* From V8 https://github.com/nodejs/node/blob/73ae2d1895c2c0d1d4eeaddd284c58d776d2be87/deps/v8/src/base/win32-headers.h#L32-L34

Reply:
(grrrr, shirt-return again)
* From uv https://github.com/nodejs/node/blob/766cd1f59d5adb03953656391e63b4186c9aff10/deps/uv/include/uv-win.h#L22-L24
* Our: https://github.com/nodejs/node/blob/a36aa049c857d87580548d096743f0acf28d231b/src/node.h#L42-L47

tl;dr this should be injected by the build system, so you should `ifndef` around it. Also semantically it's used for targeting a minimum platform requirements, and `0x0600` is anyway our platform minimum.

Reply:
I'm not at my laptop rn but if it's really just ifdef'ing it feel free to push that to this branch